src/gui/editor/Editor.ts

```diff
@@ -60,6 +60,14 @@ export class Editor implements ImageHandler, Component {
 	 * * "input" event on squire resets flag.
 	 */
 	private pasteListener: (e: ClipboardEvent) => void = (_: ClipboardEvent) => (this.userHasPasted = true)
+	private copyListener: (e: ClipboardEvent) => void = (e: ClipboardEvent) => {
+		if (e.clipboardData == null) return
+		// squire by default duplicates newlines when copying content from the editor, but .getSelectedText()
+		// contains the correct number of \n
+		// there's also another entry in the data that contains the "text/html" which is not affected by this.
+		if (!e.clipboardData.types.includes("text/plain")) return
+		e.clipboardData.setData("text/plain", this.squire.getSelectedText())
```

Comment:
is `getSelectedText` always plaintext?

Reply:
squire only walks text nodes and appends their data field (which is plain text) to an accumulator